projects/step-core/src/lib/modules/table/services/filter-condition-factory.service.ts

```diff
@@ -14,7 +14,7 @@ import { FilterConditionJson } from '../shared/filter-condition-json.interface';
 import { FilterConditionType } from '../shared/filter-condition-type.enum';
 import { BooleanFilterCondition } from '../shared/boolean-filter-condition';
 import { DateRangeFilterCondition } from '../shared/date-range-filter-condition';
-import { DateRange } from '../../date-picker/date-picker.module';
+import { DateRange } from '../../date-picker/types/date-range';
```

Comment:
```suggestion
import { DateRange } from '../../date-picker/date-picker.module';
```

Reply:
the imports really did the trick with the cycle dependencies